pkg/quarterdeck/quarterdeck_test.go

```diff
@@ -83,7 +83,7 @@ func (s *quarterdeckTestSuite) SetupSuite() {
 			RefreshOverlap:  -10 * time.Minute,
 		},
 		RateLimit: config.RateLimitConfig{
-			PerSecond: 20.00,
+			PerSecond: 60.00,
```

Comment:
Not sure if anybody else was seeing this but I got some errors in tests that do repeated calls so I upped the limit just for the test suite.